src/components/Home/Home.js

```diff
@@ -1,5 +1,79 @@
-import React from 'react';
+import React, {useEffect} from 'react';
+import SearchBar from '../SearchBar/SearchBar'
+import styles from './Home.scss'
+import { Container, Row, Col } from 'react-grid-system';
+import img from '../../assets/pictures/political.jpg'
+import Navigation from '../Navigation/Navigation'
+import NavigationBar from '../Navigation/NavigationBar';
+console.log(styles);
```

Comment:
we don't want to have `console.log`'s in our production code so let's make sure to delete them before committing your code